src/test/regress/expected/multi_mx_schema_support.out

```diff
@@ -538,7 +538,28 @@ SELECT run_command_on_workers($$DROP SCHEMA localschema;$$);
  (localhost,57638,f,"ERROR:  schema ""localschema"" does not exist")
 (2 rows)
 
+CREATE ROLE schema_owner WITH LOGIN;
+NOTICE:  not propagating CREATE ROLE/USER commands to worker nodes
+HINT:  Connect to worker nodes directly to manually create all necessary users and roles.
+SELECT run_command_on_workers($$CREATE ROLE schema_owner WITH LOGIN;$$);
+      run_command_on_workers
+---------------------------------------------------------------------
+ (localhost,57637,t,"CREATE ROLE")
+ (localhost,57638,t,"CREATE ROLE")
+(2 rows)
+
 RESET citus.enable_ddl_propagation;
+-- create schema with the name of the owner
+CREATE SCHEMA AUTHORIZATION schema_owner;
+-- verify the schema is created on workers
+SELECT run_command_on_workers($$SELECT COUNT(*) FROM pg_namespace WHERE nspname='schema_owner';$$);
```

Comment:
This query is not allowed on enterprise and causes tests failures.

```diff
 SET client_min_messages TO ERROR;
 CREATE ROLE schema_owner WITH LOGIN;
 RESET client_min_messages;
 SELECT run_command_on_workers($$CREATE ROLE schema_owner WITH LOGIN;$$);
                        run_command_on_workers                        
 ---------------------------------------------------------------------
- (localhost,57637,t,"CREATE ROLE")
- (localhost,57638,t,"CREATE ROLE")
+ (localhost,57637,f,"ERROR:  operation is not allowed on this node")
+ (localhost,57638,f,"ERROR:  operation is not allowed on this node")
 (2 rows)
```

See https://github.com/citusdata/citus-enterprise/pull/778